src/helpers/createSupplierFromCsv.js

```diff
@@ -23,6 +23,16 @@ const createSupplierFromCsv = (row) => {
     fuelMix: row["fuelMix"],
     status: PARSED,
   };
+
+  if (import.meta.env.VITE_REACT_APP_FF_UPDATE_CSR === "true") {
+    const clonedRow = { ...row };
```

Comment:
I'm not entirely sure why the row is cloned here - it's never changed in any way so we can just use row?

Reply:
I need to double check this as yesterday I ran into some issues where when I was testing the code it wasn't correctly populating the field in Contentful without using the clonedRow.  However, I'll remove it and check it again

Reply:
Well, thankfully it appears to all be working as expected this morning so I've removed the cloned row 🙂